fapolicy_analyzer/ui/subject_list.py

```diff
@@ -164,12 +197,41 @@ def on_view_row_activated(self, treeView, row, *args):
         self.__show_reconciliation_dialog(subject)
 
     def on_view_button_press_event(self, widget, event):
+        treeView = self.get_object("treeView")
+        model, pathlist = treeView.get_selection().get_selected_rows()
+        n_paths = len(pathlist)
         if event.type == Gdk.EventType.BUTTON_PRESS and event.button == 3:
-            self.contextMenu.popup_at_pointer()
+            if n_paths == 1:
+                self.reconcileContextMenu.popup_at_pointer()
+            elif n_paths > 1:
+                trust = list(
+                    {
+                        self.find_db_trust(model.get_value(model.get_iter(path), 3))
+                        for path in iter(pathlist)
+                    }
+                )
+                if len(trust) > 1:
+                    return
+
+                self.fileChangeContextMenu.remove(self.fileChangeContextMenu.trustItem)
+                self.fileChangeContextMenu.remove(self.fileChangeContextMenu.untrustItem)
+                if isinstance(trust[0], Trust):
+                    self.fileChangeContextMenu.append(self.fileChangeContextMenu.untrustItem)
+                else:
+                    self.fileChangeContextMenu.append(self.fileChangeContextMenu.trustItem)
```

Comment:
Similar to up in line 169, you need to use both the `subject.trust` and the `dbtrust.status` values here to determine what menu option to show.